test/compress/nullish.js

```diff
@@ -0,0 +1,31 @@
+parenthesis: {
```

Comment:
spelling - parentheses

Reply:
Fixed in https://github.com/mishoo/UglifyJS/pull/4678/commits/30f57281a4ce3e869e8ac86031d4b8713f63c5a1